tests/lax_test.py

```diff
@@ -69,6 +67,27 @@
   (np.int32, np.float32), (np.int32, np.float64), (np.int64, np.float64)]
 
 
+def _reduce_custom_add(x, y):
+  return x + y
+
+def _reduce_custom_mul(x, y):
+  return x * y
+
+def _reduce_custom_sub(x, y):
+  return x - y
```

Comment:
Can we just use `operator.add` etc for these, instead of defining new functions?

Reply:
Maybe not, if the lax code actually detects operator.add specifically...